### DISCUSSION TITLE
Starting All Queries gives error : This type of CollectionView does not support changes to its SourceCollection from a thread different from the Dispatcher thread

### DISCUSSION THREAD

**Discussion by infostreambe** [DaxStudio/DaxStudio#1304] [General]
Hello,

When I start the All Queries option, the error "This type of CollectionView does not support changes to its SourceCollection from a thread different from the Dispatcher thread" is generated in the log file and the trace stops.
Are others seeing this error as well ?

I'm running DAX 3.2.0 and Power BI Desktop August 2024.

**Comment by infostreambe** [comment]
Hello, I just installed DAX Studio 3.2.1 and All Queries is working again.
Thank you for the quick fix !

**Comment by bluehenve** [comment]
Hi
I also get this error in DAX Studio 3.2.1, rigth when starting All Queries.
DAX Studio is connected to Power BI desktop, with a few SQL tables importet to import mode

**Reply by bluehenve** [reply]
Solved: Just have to start Dax Studio as administrator.
Is is NOT enough to start Power BI desktop as admin an then open Dax studio through External tools > Dax Studio
